src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="search"
+      data-qa="big"
+    >
+      <div class="search__block">
```

Comment:
```suggestion
      <label class="search__block">
```
its better to use `label` here